scapy/pipetool.py

```diff
@@ -474,26 +497,51 @@ def __init__(self, name=None, keepterm=True, newlines=True, openearly=True):
     def start(self):
         if not self.opened:
             self.opened = True
-            self.__r,self.__w = os.pipe()
-            cmd = ["xterm"]
-            if self.name is not None:
-                cmd.extend(["-title",self.name])
-            if self.keepterm:
-                cmd.append("-hold")
-            cmd.extend(["-e", "cat 0<&%i" % self.__r])
-            self.__p = subprocess.Popen(cmd)
-            os.close(self.__r)
+            if WINDOWS:
+                self.__f = get_temp_file()
+                self.name = "Scapy" if self.name is None else self.name
+                # Start a powershell in a new window and print the PID
+                cmd = "$app = Start-Process PowerShell -ArgumentList '-command &{$host.ui.RawUI.WindowTitle=\\\"%s\\\";Get-Content \\\"%s\\\" -wait}' -passthru; echo $app.Id" % (self.name, self.__f.replace("\\", "\\\\"))
```

Comment:
The goal of the program was to start a new shell on the computer to print some stuff. To emulate `tail -f`, I use Get-Content, only available in Powershell.